tests/features/piwik.feature

```diff
@@ -1,20 +1,89 @@
-@wip
+@wip @api @reset-nodes
 Feature: Check Piwik
   In order to check if the the type attribute is set for the Piwik element.
   As an administrator
   I want to check Piwik is available.
+  # Advanced PIWIK rules functionality
+  In order to make PIWIK analysis more granular and accurate
+  As a site administrator
+  I can define advanced PIWIK rules to define the site section based on path or regular expression
 
-@api
-Scenario: Administrator user can check Piwik Script with the theme Bootstrap
-  Given the module is enabled
-    | modules            |
-    | nexteuropa_piwik   |
-  Given I am logged in as a user with the 'administrator' role
-  When I run drush "pm-enable bootstrap -y"
-  And I run drush "vset theme_default bootstrap"
-  Then I am on the homepage
-  And the response should contain "{\"utility\":\"piwik\",\"siteID\":\"\",\"sitePath\":[\"\"],\"is404\":false,\"instance\":\"\"}"
-  When I run drush "vset theme_default ec_resp"
-  Then I am on the homepage
-  And the response should contain "{\"utility\":\"piwik\",\"siteID\":\"\",\"sitePath\":[\"\"],\"is404\":false,\"instance\":\"\"}"
+  Background:
+    Given these modules are enabled
+      | modules            |
+      | nexteuropa_piwik   |
+    And the nexteuropa_piwik module is configured to use advanced PIWIK rules
+    And I am logged in as a user with the "administrator" role
 
+  @wip
+  Scenario: Administrator user can check Piwik Script with the theme Bootstrap
+    Given the module is enabled
```

Comment:
Indeed but I've kept the old scenario which was previously implemented for the PIWIK module.